docs/cody/proxy-setup.mdx

```diff
@@ -0,0 +1,112 @@
+
+# Proxy Setup Guide
+
+## Introduction
+This guide will walk you through setting up HTTP, HTTPS, and SOCKS proxies in Visual Studio Code (VS Code) and JetBrains IDEs. It also includes instructions for handling self-signed certificates on macOS and Windows.
+
+## Proxy Setup for VS Code
+
```

Comment:
@MaedahBatool It makes sense to me to do the simple tabs here for VSCode vs JBs. LMK what you think? Not a blocker, just a simple readability improvement